selectors.yml

```
@@ -33,10 +39,13 @@ selectors:
   2358:
     selector: 5990477251245693094
     name: "ethereum-testnet-sepolia-kroma-1"
-  4002: 
+  2522:
+    selector: 8901520481741771655
+    name: "fraxtal-testnet-optimism-1"
```

This name indicates this is the first instance of a `optimism` (like, official) rollup being deployed to a unnamed/first/main testnet for the `fraxtal` L1. I was not aware of Optimistic Labs deploying official instances to these other L1s? Can you please provide more information to this network, and make sure it complies with the [naming pattern](https://www.notion.so/chainlink/Network-naming-proposal-161255913454455c9ba01f5cc25e5379)?

---

From what I can see from their site, they're a L2, anchored to Ethereum as its L1. Optimism is just the technology.
Their testnet is attached to Ethereum's [Holesky](https://holesky.fraxscan.com/) testnet.
Therefore, the naming should be:
```suggestion
    name: "ethereum-testnet-holesky-fraxtal-1"
```

---

Got it thank you! Updated.